src/seedu/addressbook/data/person/Address.java

```diff
@@ -6,53 +6,18 @@
  * Represents a Person's address in the address book.
  * Guarantees: immutable; is valid as declared in {@link #isValidAddress(String)}
  */
-public class Address {
+public class Address extends Contact {
 
     public static final String EXAMPLE = "123, some street";
     public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
     public static final String ADDRESS_VALIDATION_REGEX = ".+";
 
```

Comment:
No, I should have made it private instead as it  is the child class from of the parent class, Contact. 

> On 9 Sep 2016, at 12:08, KulaniT <notifications@github.com> wrote:
> 
> In src/seedu/addressbook/data/person/Address.java:
> 
> > @@ -6,53 +6,18 @@
> >   * Represents a Person's address in the address book.
> >   * Guarantees: immutable; is valid as declared in {@link #isValidAddress(String)}
> >   */
> > -public class Address {
> > +public class Address extends Contact {
> >  
> >      public static final String EXAMPLE = "123, some street";
> >      public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
> >      public static final String ADDRESS_VALIDATION_REGEX = ".+";
> >  
> Think. Do you need these to be public?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>